src/openms/include/OpenMS/ANALYSIS/ID/AScore.h

```diff
@@ -77,29 +78,32 @@ namespace OpenMS
 
         @note the original sequence is saved in the PeptideHits as MetaValue Search_engine_sequence.
     */
-    PeptideHit compute(PeptideHit & hit, RichPeakSpectrum & real_spectrum, double fmt, Int number_of_phospho_sites);
+    PeptideHit compute(PeptideHit & hit, RichPeakSpectrum &real_spectrum, double fragment_mass_tolerance, bool fragment_mass_unit_ppm);
+
+    ///Computes the site determining_ions for the given AS and sequences in candidates
+    void computeSiteDeterminingIons(std::vector<RichPeakSpectrum> & th_spectra, ProbablePhosphoSites & candidates, Int charge, std::vector<RichPeakSpectrum> & site_determining_ions);
 
-    ///Computes the cumulative binomial probabilities.
-    double computeCumulativeScore(UInt N, UInt n, double p);
+    /// return all phospho sites
+    std::vector<Size> getSites(AASequence & without_phospho);
```

Comment:
most methods are not const (I have not looked into the body yet).
specific reason?